rubygem-foreman_pipeline/rubygem-foreman_pipeline.spec

```diff
@@ -11,54 +11,39 @@
 
 %global gem_name foreman_pipeline
 %global foreman_dir /usr/share/foreman
-%global scl_rake /usr/bin/%{?scl:%{scl_prefix}}rake
+%global foreman_bundlerd_dir %{foreman_dir}/bundler.d
 
-%define rubyabi 1.9.1
+%global scl_rake /usr/bin/%{?scl:%{scl_prefix}}rake
+%global scl_ruby_bin /usr/bin/%{scl:%{scl_prefix}}ruby
```

Comment:
This shouldn't be needed as the macro handles it. Do you get errors if you don't define it here?